### DISCUSSION TITLE
Move to display where cursor (optional by checkbox)

### DISCUSSION THREAD

**Discussion by porg** [rxhanson/Rectangle#1397] [Ideas]
## Proposal: The third settings screen gets this new setting

`[√] Move to display where cursor`

- If activated all move commands will move the focused window to its target position on the display where the cursor currently resides.

## Operation
1) Focus the window you want to move.
2) Optional: Move mouse to other display where the window shall get positioned to.
   Note: If you keep your cursor where it is, then the window will move within the same display.
3) Trigger window move command.

## Usability Rationale

- For efficient multi display window positioning this workflow is ultra efficient.

## Best Practise

- [Divvy](https://mizage.com/divvy/) does just that by default (without a preference).
- Served me well for years, but I move from an Intel Mac to Apple Silicon now.
- Divvy looks abandoned, no Apple Silicon version is offered on the website.

## The current workarounds for step 2 in Rectangle

- Trigger shortcut for "Next Display". Note: On a 2 monitor setup this effectively is "Toggle Display", so this itself needs no further thought.

OR

- Move window to other display by dragging

But with both of these it does not feel so smooth, produces more mental load. Whereas with the proposed workflow it feels very natural and undisruptive , your eyes and trackpad move in concert to your intended display and there you press your intended position hotkey and are done in one pass.

**Comment by rxhanson** [comment]
Thank you for the detailed feedback. I don't have the bandwidth to add this one in myself at this time, but would welcome a good pull request for it.

**Reply by porg** [reply]
I'm a UX designer by profession, and at best a scripter, but no developer.

If anyone does a pull request, please ping me here, I'm ready to do beta testing and quality assurance!
